### PR TITLE
Added all API Routes

### LINKED ISSUE #34: update all API Routes in index.js
create and update all API routes in index .html

### PR DESCRIPTION
**Pull request by malivinayak**
## API ROUTES

1. registration APIs
2. login APIs
3. session-end APIs
4. credentials APIs
5. admin query APIs
6. trip APIs
7. Wallet API's

Closes #34

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -26,4 +44,27 @@ app.post("/api/registration/user", userRegistration);
...
+// trip APIs
+app.post("/api/trip/booking", tripBooking);
+app.post("/api/trip/history/:role", tripHistory);
+app.post("/api/trip/available", tripAvailable);
+app.post("/api/trip/accepting", tripAccepting);
+app.post("/api/trip/rating", tripRating);
+
+//Wallet APIs
+app.post("/api/wallet/addMoney", addMoney);
+app.post("/api/wallet/withdrawMoney", withdrawMoney);
```
Comment (OutdatedGuy) [inline src/index.js]: And new line at end

File: src/index.js [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+import { tripAccepting } from "./backend/routes/trip/tripAccepting.js";
+import { tripRating } from "./backend/routes/trip/tripRating.js";
+// Admin Queries
+import { userQuery } from "./backend/routes/queries/userQuery.js";
+import { driverQuery } from "./backend/routes/queries/driverQuery.js";
+import { tripQuery } from "./backend/routes/queries/tripQuery.js";
+import { customQuery } from "./backend/routes/queries/customQuery.js";
+// Wallet
+import { addMoney } from "./backend/routes/wallet/addMoney.js";
+import { withdrawMoney } from "./backend/routes/wallet/withdrawMoney.js";
```
Comment (OutdatedGuy) [inline src/index.js]: Added newline between different sections

**Review by OutdatedGuy — CHANGES_REQUESTED** [review]

**Comment by malivinayak** [comment]
I have [added new line at the end](https://github.com/malivinayak/Trip-Management-System/pull/36/commits/9c95f6a796678eb10864bb2c88b09b570a4c0ee6) in this commit
but it is not showing here
image

**Review by OutdatedGuy — APPROVED** [review]
LGTM!

**Comment by OutdatedGuy** [comment]
> I have [added new line at the end](https://github.com/malivinayak/Trip-Management-System/pull/36/commits/9c95f6a796678eb10864bb2c88b09b570a4c0ee6) in this commit but it is not showing here image

If at the end it shows this symbol 
image
 means there not newline.

**Comment by malivinayak** [comment]
> > I have [added new line at the end](https://github.com/malivinayak/Trip-Management-System/pull/36/commits/9c95f6a796678eb10864bb2c88b09b570a4c0ee6) in this commit but it is not showing here image
> 
> If at the end it shows this symbol image means there not newline.

Got IT

**Pull request merged**